### PR TITLE
#162163137 Pagination support for articles

### PR DESCRIPTION
**Pull request by lindseyme**
### What does this PR do?
This Pull Request adds pagination support to articles.

#### Description of Task to be completed?
Add pagination to our articles to make article traversal easier.

#### How should this be manually tested?
- Clone the repository then checkout to the branch `ft-pagination-162163137`.
- Install the required dependencies.
     - yarn 
- Start the server.
     - yarn start
- Run the project in the browser.
- Click on the login button on the interface and fill in the details. 
- The articles are displayed on pages and a maximum of 10 per page.

#### What are the relevant pivotal tracker stories?
[#162163137](https://www.pivotaltracker.com/story/show/162163137)

### CODE REVIEW

File: src/components/articles/ArticlesFeed.js [hunk 1]
```diff
@@ -5,8 +5,13 @@ import { Link } from 'react-router-dom';
...
 
 export class Articles extends Component {
+  constructor(props) {
+    super(props);
+    this.next = this.next.bind(this);
+  }
+
   componentDidMount() {
-    this.props.getArticles();
+    this.props.getArticles('https://zeus-staging.herokuapp.com/api/articles/');
```
Comment (sekayasin) [inline src/components/articles/ArticlesFeed.js]: Bravo, just change this to our environment variable

**Review by ekumamait — APPROVED** [review]
LQTM

File: src/components/articles/ArticlesFeed.js (hunk 1 above)
Comment (lindseyme) [reply src/components/articles/ArticlesFeed.js]: Okay, I have done that

**Review by ongebo — APPROVED** [review]
LGTM

**Review by barema4 — APPROVED** [review]
Nice work

**Review by RrNn — CHANGES_REQUESTED** [review]
@lindseyme , the `Next` button should be on the right.
screen shot 2019-01-30 at 11 17 53

**Comment by lindseyme** [comment]
> @lindseyme , the `Next` button should be on the right.
> screen shot 2019-01-30 at 11 17 53

**Pull request merged**